src/main/java/io/gravitee/repository/jdbc/management/JdbcParameterRepository.java

```diff
@@ -144,14 +153,14 @@ public void delete(String key, String referenceId, ParameterReferenceType refere
             if (isEmpty(keys)) {
                 return Collections.emptyList();
             }
-            List<Parameter> parameters = jdbcTemplate.query("select * from parameters where reference_id = ? and reference_type = ? and " + escapeReservedWord("key") + " in ( "
-                            + ORM.buildInClause(keys) + " )"
+            List<Parameter> parameters = jdbcTemplate.query(getOrm().getSelectAllSql() + " where reference_id = ? and reference_type = ? and " + escapeReservedWord("key") + " in ( "

```

Comment:
getSelectAll() escapes the table name.

```suggestion
            List<Parameter> parameters = jdbcTemplate.query("select * from " + this.tableName + " where reference_id = ? and reference_type = ? and " + escapeReservedWord("key") + " in ( "

```